alcs-api/src/application/application.controller.ts

```diff
@@ -67,10 +71,15 @@ export class ApplicationController {
     @Body() application: CreateApplicationDto,
   ): Promise<ApplicationDto> {
     const type = await this.applicationTypeService.get(application.type);
+    const decisionMaker = application.decisionMaker
+      ? await this.applicationDecisionMakerService.get(
+          application.decisionMaker,
+        )
+      : undefined;
     const app = await this.applicationService.createOrUpdate({
       ...application,
       type,
-      typeUuid: type.uuid,
+      decisionMaker,
       title: 'Do we still need title?',
```

Comment:
Did we ever get response on this one or we waiting for wireframes?

Reply:
None yet, I've gone ahead and deleted it and replace the title with the same as the card fileNumber (applicant)